containers/docker-packager/Dockerfile

```
@@ -28,9 +28,8 @@ COPY containers/docker-packager/registries.conf \
      containers/docker-packager/storage.conf \
      /etc/containers/
 
-COPY packagers/docker/requirements.txt "${WORK_DIR}/packagers/docker/requirements.txt"
+COPY ./packagers packagers/
 RUN pip3 install --upgrade pip setuptools wheel && \
-    pip3 install -r "${WORK_DIR}/packagers/docker/requirements.txt"
+    pip3 install -r packagers/requirements.txt
 
-COPY packagers/docker "${WORK_DIR}/packagers/docker"
-RUN pip3 install -e packagers/docker
+RUN pip3 install -e packagers/
```

```suggestion
RUN python -m pip install --user -e packagers/
```

As I was given the WARNINGs:
> #10 0.680 WARNING: Running pip install with root privileges is generally not a good idea. Try `pip3 install --user` instead.


> #10 9.509 WARNING: pip is being invoked by an old script wrapper. This will fail in a future version of pip.
#10 9.509 Please see https://github.com/pypa/pip/issues/5599 for advice on fixing the underlying issue.
#10 9.509 To avoid this problem you can invoke Python with '-m pip' instead of running pip directly.

it's also the suggestion for the previous `RUN`, I just couldn't select them together.